platforms/posix/src/px4/common/px4-alias.sh_in

```diff
@@ -2,6 +2,8 @@
 
 # File is auto-generated by cmake compilation, do not edit.
 
+R=${pwd}
```

Comment:
```suggestion
R="$(pwd)/"
```
And replace all `sh x` with `. ${R}x`. Then it should work.

---

Alternatively we could do:
```
sh_exec() {
	. $(pwd)/$1
}
sh=sh_exec

# then use it as:
${sh} etc/init.d/rc.mc_defaults
```
And on Nuttx `set sh .` if that works.

Reply:
This did not work. However R="`pwd`/" does.